spec/models/onboard_judging_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'rails_helper'
 
-RSpec.describe OnboardJudging, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe OnboardJudging do
+  describe 'score' do
+    context 'negative onboard judging' do
+      let(:onboard_judging) { create :onboard_judging, minutes_elapsed: 11 }
+      it 'score should be negative' do
```

Comment:
I think more to the point is that it `allows negative scores`, unlike the other models which don't.